tests/integrationtest/r/planner/core/casetest/physicalplantest/physical_plan.result

```diff
@@ -3836,12 +3836,12 @@ explain format = 'brief' select min(col_304) from (select /*+ use_index_merge( t
 id	estRows	task	access object	operator info
 StreamAgg	1.00	root		funcs:min(planner__core__casetest__physicalplantest__physical_plan.tbl_43.col_304)->Column#2
 └─IndexMerge	1.00	root		type: union, limit embedded(offset:0, count:1)
-  ├─Limit(Build)	0.00	cop[tikv]		offset:0, count:1
-  │ └─TableRangeScan	0.00	cop[tikv]	table:tbl_43	range:["LUBGzGMA","LUBGzGMA"], keep order:true, stats:pseudo
   ├─Limit(Build)	0.33	cop[tikv]		offset:0, count:1
-  │ └─IndexRangeScan	0.33	cop[tikv]	table:tbl_43, index:idx_261(col_304)	range:[-inf,"YEpfYfPVvhMlHGHSMKm"), keep order:true, stats:pseudo
+  │ └─TableRangeScan	0.33	cop[tikv]	table:tbl_43	range:[-inf,"YEpfYfPVvhMlHGHSMKm"), keep order:true, stats:pseudo
```

Comment:
why this range change from a point of “LUBGzGMA” to a half-open one

Reply:
Actually it's the order of partial paths that changes.

Before:
```
TableRangeScan  ["LUBGzGMA","LUBGzGMA"]
IndexRangeScan  [-inf,"YEpfYfPVvhMlHGHSMKm")
IndexRangeScan  ("PE",+inf]
```
After:
```
IndexRangeScan  [-inf,"YEpfYfPVvhMlHGHSMKm")
IndexRangeScan  ("PE",+inf]
TableRangeScan  ["LUBGzGMA","LUBGzGMA"]
```

Reply:
got